src/commands/uptime.ts

```diff
@@ -0,0 +1,18 @@
+import type { Command } from "../command";
+
+const command: Command = {
+	name: "uptime",
+	description: "Displays bot uptime.",
+	options: [],
+	hasPermission: () => true,
+	shouldBeEphemeral: (state, interaction) => interaction.channelID !== state.config.botChannelId,
+	handler: (state, interaction) => {
+		interaction
+			.reply(`Uptime: ${Math.floor(process.uptime())} seconds.`, {
```

Comment:
Should open an issue to convert this from seconds into a more comprehensible format.

Reply:
#59